src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/SARIF1009.IndexPropertiesMustBeConsistentWithArrays_Invalid.sarif

```diff
@@ -123,7 +123,7 @@
                   "index": 0
                 },
                 "region": {
-                  "startLine": 154,
+                  "startLine": 155,
```

Comment:
I have looked,
the 155 line is the correct line that is 
`"index": 2`
so the after change is correct.

Reply:
Please figure out why this changed.

Reply:
still looking this one